components/budget/filters/PeriodFilter.js

```diff
@@ -154,8 +154,52 @@ const PeriodFilter = ({ onChange, value, inputId, minDate, ...props }) => {
               interval={tmpDateInterval}
             />
           </Box>
+          <StyledInputField
+            name="timezoneType"
+            mt="12px"
+            labelFontSize="13px"
```

Comment:
According to design; 

```suggestion
            labelFontSize="16px"
```